### PR TITLE
Blog App

### PR DESCRIPTION
**Pull request by david-lafontant**
**Budget App is a mobile web application that helps users manage expenses.**

_Budget App allows the user to:_
- [x] register and log in, so that the data is private to them.
- [x] introduce new transactions associated with a category.
- [x] see the money spent on each category.

_Budget App:_

- [x] Implements [project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Project%20requirements,a%20small%20picture.)
- [x] Has no linter errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,83 @@
...
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse for giving me this chance
+- The design template provider [Gregoire Vella on Behance](https://www.behance.net/gregoirevella)(Sanpscan App).
+
+## 📝 License
+
+This project is [MIT](./LICENCE) licensed.
```
Comment (Rustamxon7) [inline README.md:83]: - [ ] [OPTIONAL] You forgot to add an MIT license to your project. Kindly make sure your app must contain an MIT license in it.

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h1>Users#index</h1>
```
Comment (Rustamxon7) [inline app/views/users/index.html.erb:1]: - [ ] [OPTIONAL] You can delete unused files from your project.

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class UsersController < ApplicationController
```
Comment (Rustamxon7) [inline app/controllers/users_controller.rb:1]: - [ ] [OPTIONAL] You can delete unused files from your project.

File: app/controllers/pages_controller.rb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+class PagesController < ApplicationController
+  def index; end
```
Comment (Rustamxon7) [inline app/controllers/pages_controller.rb:2]: - [ ] [OPTIONAL] You can delete unused files from your project.

File: app/controllers/groups_controller.rb [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    @group.user_id = current_user.id
+    if @group.save
+      flash[:success] = 'Group created succesfully'
+      redirect_to user_groups_path(current_user)
+    else
+      render 'new'
+    end
+  end
+
+  def show; end
```
Comment (Rustamxon7) [inline app/controllers/groups_controller.rb:21]: - [ ] [OPTIONAL] You can delete unused files/lines of code from your project.

**Review by rustam-tolipov — APPROVED** [review]
## Hi @david-lafontant👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by david-lafontant** [comment]
@Rustamxon7 thank you very much for your time and feedback. I will implement your recommendation with the enhancement that I plan to implement. I wish to add a menu (a classic mobile "hamburger" menu and some animation. I'm learning now how to make it. Again thank you and have a great day. 
Regards

**Pull request merged**